tests/unit/amo/pages/TestHome.js

```diff
@@ -541,14 +542,15 @@ describe(__filename, () => {
     const heroShelves = _createHeroShelves();
     _loadHomeData({ store, heroShelves });
     const root = render({ store });
+    const lang = config.get('defaultLang');
```

Comment:
Instead of importing `config` above and calling it here, you can import `DEFAULT_LANG_IN_TESTS` from `tests/unit/helpers` and use that.

Reply:
https://github.com/mozilla/addons-frontend/pull/10086/commits/7d92b37c782685e84da506439bf1f1cf070b5806